modules/nodes-cluster-worker-latency-profiles-about.adoc

```diff
@@ -49,21 +51,21 @@ The Kubernetes Controller Manager Operator waits 40 seconds for a status update
 | `node-monitor-grace-period`
 | 40s
 
-| Kubernetes API Server
+| Kubernetes API Server Operator
```

Comment:
Based on the qe doc we were sent about how to validate these numbers, this should just be Kubernetes API Server

Reply:
@paigerube14 I'll remove the word "Operator" per the doc.